packages/react/src/components/Modal/index.tsx

```diff
@@ -94,26 +94,28 @@ export const ModalInner = ({ closeOnEsc = true, ...props }: ModalProps) => {
       shouldCloseOnEsc={closeOnEsc}
       bodyOpenClassName={classNames("c__modals--opened", NOSCROLL_CLASS)}
     >
-      {!props.hideCloseButton && !props.preventClose && (
-        <div className="c__modal__close">
-          <Button
-            icon={<span className="material-icons">close</span>}
-            color="tertiary-text"
-            size="small"
-            onClick={() => props.onClose()}
-          />
+      <div className="c__modal__scroller">
+        {!props.hideCloseButton && !props.preventClose && (
+          <div className="c__modal__close">
+            <Button
+              icon={<span className="material-icons">close</span>}
+              color="tertiary-text"
+              size="small"
+              onClick={() => props.onClose()}
```

Comment:
Shorter and prevent to create uselessly a lambda function
```suggestion
              onClick={props.onClose}
```